lib/Runtime/Library/JavascriptSimdType.cpp

```diff
@@ -26,9 +26,9 @@ namespace Js
 
         if (args.Info.Count == 0 || !SIMDType::Is(args[0]))
         {
-            char16 buffer[512] = _u("");
-            char16* bufferPtr = buffer;
-            JavascriptError::ThrowTypeError(scriptContext, JSERR_This_NeedSimd, SIMDType::GetFullBuiltinName(&bufferPtr, _u("toString()")));
+            char16 buffer[SIMD_STRING_BUFFER_MAX] = _u("");
+            swprintf_s(buffer, SIMD_STRING_BUFFER_MAX, SIMDType::GetTypeName(), _u(".toString()"));
```

Comment:
These GetTypeName functions return a string, not a format string.  The .toString() won't get appended anymore.